content/en/observability_pipelines/setup/splunk.md

```diff
@@ -0,0 +1,347 @@
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+To run the Worker on your Kubernetes nodes, you need a minimum of two nodes with one CPU and 512MB RAM available. Datadog recommends creating a separate node pool for the Workers, which is also the recommended configuration for production deployments.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+To run the Worker on your Kubernetes nodes, you need a minimum of two nodes with one CPU and 512MB RAM available. Datadog recommends creating a separate node pool for the Workers, which is also the recommended configuration for production deployments.
+{{% /tab %}}
+{{% tab "APT-based Linux" %}}
+There are no provider-specific requirements for APT-based Linux.
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+There are no provider-specific requirements for RPM-based Linux.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Setting up the Splunk index
+
+<div class="alert alert-info">Observability Pipelines supports acknowledgements when you enable the <strong>Enable Indexer Acknowledgements</strong> setting on the input.</div>
+In order to receive logs from the Observability Pipelines Worker, you will need to provision a HEC input and HEC token on the index.
+
+Under **Settings > Data Inputs**, add a new HTTP Event Collector input. You can name it whatever you wish.
+
+{{< callout url="#" btn_hidden="true" header="Acknowledgements" >}}
+  Observability Pipelines supports acknolwedgements when you enable the <strong>Enable Indexer Acknowledgements</strong> setting on this input.
+{{< /callout >}}
+
+Select whichever index(es) you wish for logs to flow into.
+
+Once the input is created, Splunk will create a token for you, usually in a UUID format. In the sample configurations below, you will add this token so that the Observability Pipelines Worker can authenticate itself.
+
+## Installing the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+1. Download the [Helm chart][1] for AWS EKS.
+
+2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline. Then, replace the values for `SPLUNK_ENDPOINT` and `SPLUNK_HEC_TOKEN` to match your Splunk deployment (and the token you created above). Then, install it in your cluster with the following commands:
+
+    ```shell
+    helm repo add datadog https://helm.datadoghq.com
+    ```
+    ```shell
+    helm repo update
+    ```
+    ```shell
+    helm upgrade --install \
+        opw datadog/observability-pipelines-worker \
+        -f aws_eks.yaml
+    ```
+
+[1]: /resources/yaml/observability_pipelines/splunk/aws_eks.yaml
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+1. Download the [Helm chart][1] for Azure AKS.
+
+2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline. Then, replace the values for `SPLUNK_ENDPOINT` and `SPLUNK_HEC_TOKEN` to match your Splunk deployment (and the token you created above). Then, install it in your cluster with the following commands:
```

Comment:
Could we provide a code snippet that shows the code block where the user will be inserting their api key, pipeline ID, splunk endpoint & hec token? I know it seems obvious since it's at the top of the downloaded yaml, but we've seen some customers who need very direct hand holding.

Something like:

2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline:
```
datadog:
  apiKey: "<datadog_api_key>"
  pipelineId: "<observability_pipelines_configuration_id>"
  site: "datadoghq.com"
```
3. Then, replace the values for `SPLUNK_ENDPOINT` and `SPLUNK_HEC_TOKEN` to match your Splunk deployment, including the token you created in [Setting up the Splunk index](setting-up-the-splunk-index):
```
env:
  - name: SPLUNK_HEC_ENDPOINT
    value: <https://your.splunk.index:8088/>
  - name: SPLUNK_TOKEN
    value: <a_random_token_usually_a_uuid>
```
4. Then, install the observability pipelines worker in your cluster with the following commands, passing in the your `aws_eks.yaml` into the final command:
    ```shell
    helm repo add datadog https://helm.datadoghq.com
    ```
    ```shell
    helm repo update
    ```
    ```shell
    helm upgrade --install \
        opw datadog/observability-pipelines-worker \
        -f aws_eks.yaml
    ```

Reply:
I've updated the guide to add this, good idea.